### DISCUSSION TITLE
Can't import the named export 'bestPositionOf'

### DISCUSSION THREAD

**Discussion by sorojas-technisys** [elrumordelaluz/reactour#497] [General]
Hi guys,

We are compiling our app and we get this error in "yarn start" related with reacttour: 
./node_modules/@reactour/popover/dist/index.mjs
 Can't import the named export 'bestPositionOf' from non EcmaScript module (only default export is available)

We are using: node v14.14.0, npm v6.14.16, and reacttour v2.10.2.

Do you have any idea about this error??

Regards,
Sonia

**Comment by elrumordelaluz** [comment]
Thank you Sonia for this question. We are having issues with one package, where some of this helpers lives.

Please keep an eye on #498 in case they share the `npm` resolution workaround to solve this fast temporarily.
Thanks for the patience,
